blocks/product-recommendations/product-recommendations.js

```diff
@@ -5,6 +5,9 @@ import { readBlockConfig } from '../../scripts/aem.js';
 import { performCatalogServiceQuery } from '../../scripts/commerce.js';
 import { getConfigValue } from '../../scripts/configs.js';
 
+// initialize dropins
+import '../../scripts/initializers/cart.js';
```

Comment:
Would it still be possible to delay the initialization of the cart dropin here until a shopper actually clicks the add to cart button on one of the products?

Reply:
I think what you are referring to is the **_Creating of a Guest Cart_**.  However, we need to initialize the Cart drop-in to determine whether the user has a Cart and show the count indicator in the header, etc. If the user does not have a cart (missing cookie), this fetch is completely skipped by the Cart initializer (during initialization), and a new Cart only gets created when the user adds something to it.

Reply:
If this is the case, why is the cart dropin not initialized globally, like the auth dropin? :)